scripts/liquidation.js

```diff
@@ -51,14 +51,17 @@ async function main() {
 
   // Get exchange rates for oXCN and XCN
   const oXCNExchangeRate = await oXCN.callStatic.exchangeRateCurrent();
-  console.log(`1 XCN in oXCN: ${formatNumber((1 / (oXCNExchangeRate / 1e28)))}`);
-  console.log(`1 oXCN in XCN: ${formatNumber(oXCNExchangeRate / 1e28)}`);
+  const oXCNExchangeRateUnscaled = oXCNExchangeRate / 1e28;
+  console.log(`1 XCN in oXCN: ${formatNumber((1 / oXCNExchangeRateUnscaled))}`);
+  console.log(`1 oXCN in XCN: ${formatNumber(oXCNExchangeRateUnscaled)}`);
 
   // The percent, ranging from 0% to 100%, of a liquidatable account's borrow that can be repaid in a single liquidate transaction.
   // https://docs.onyx.org/comptroller/close-factor
   const closeFactorPercentage = (await Comptroller.callStatic.closeFactorMantissa() / 1e16)
+  const liquidationIncentiveMantissa = (await Comptroller.callStatic.liquidationIncentiveMantissa() / 1e16);
+  const liquidationIncentiveCoefficient = (await Comptroller.callStatic.liquidationIncentiveMantissa() / 1e18);
```

Comment:
Using additional variable to keep formulas simple, because:
`liquidationIncentiveCoefficient = (await Comptroller.callStatic.liquidationIncentiveMantissa() / 1e16); // 110, whilst we need 1.1 for calculations`

In javascript:
`1.1 / 1e2 = 0.011000000000000001`